.travis.yml

```diff
@@ -13,19 +13,25 @@ matrix:
       language: go
       go: 1.12.x
       env: GO111MODULE=on # needed even for Go 1.12
-      script: go test -v ./...
+      script:
+        go test -coverprofile=coverage.out -v ./...
+        go tool cover -func=coverage.out
```

Comment:
This ends up as a multiline script, not as two separate entries. I think you need to prefix the lines with a dash.

At least that is how I interpret the travis logs.

Reply:
Yeah. I love yaml :)